### PR TITLE
Added verbatim option for #221

### PR DESCRIPTION
**Pull request by RomanTsegelskyi**
Added `emphasize.verbatim.*` as suggested in #221. Updated documentation, vignette and examples accordingly.

```r
> pander(head(iris), emphasize.strong.cols = 2:3, emphasize.verbatim.cols = 2)

-------------------------------------------------------------------
 Sepal.Length   Sepal.Width   Petal.Length   Petal.Width   Species 
-------------- ------------- -------------- ------------- ---------
     5.1         **`3.5`**      **1.4**          0.2       setosa  

     4.9          **`3`**       **1.4**          0.2       setosa  

     4.7         **`3.2`**      **1.3**          0.2       setosa  

     4.6         **`3.1`**      **1.5**          0.2       setosa  

      5          **`3.6`**      **1.4**          0.2       setosa  

     5.4         **`3.9`**      **1.7**          0.4       setosa  
-------------------------------------------------------------------
```

### CODE REVIEW

File: R/helpers.R [hunk 1]
```diff
@@ -253,13 +253,19 @@ emphasize.rows <- function(x)
 #' @export
 emphasize.strong.rows <- emphasize.rows
 #' @export
+emphasize.italics.rows <- emphasize.rows
```
Comment (daroczig) [inline R/helpers.R]: Can we please have the same helpers for `verbatim` as well?

**Comment by daroczig** [comment]
Other then that, looks good, thanks a lot.

**Comment by daroczig** [comment]
@RomanTsegelskyi, not sure if you run `roxygen` after the last commit -- the `NAMESPACE` file is not updated.

**Comment by RomanTsegelskyi** [comment]
@daroczig, it seems like I forgot to add it to commit. I will add that in the next commit

**Pull request merged**